applications/java/functest/javaFunctest.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+echo "This script is performing the Funnctional test for Java application"
+
+echo "Based on the input parameter, color marked will be deemded as functionality failure"
+
+echo "The number of arguments is: $#"
+a=$#
+if [ $a -ne 2 ]
+then
+ 
+echo "Two parameters are expected"
+exit 1
+
+fi
+
+clusterIP=$1
+failedColor=$2
+
+curl -s "http://${clusterIP}/java-app/" > /tmp/curlout
+ if grep -q -i "$failedColor" /tmp/curlout
```

Comment:
let's reverse and fail of the color returned by the app does not match the expected color passed as argument

Reply:
done